src/index.ts

```diff
@@ -1 +1,8 @@
-console.info("hey");
+import { Client, Intents } from "discord.js"; 
+import config from "dotenv";
+config.config();
```

Comment:
```suggestion
import "dotenv/config"

```There should be a new line after the imports and importing `dotenv/config` does the same thing